{{cookiecutter.project_name}}/Jenkinsfile

```diff
@@ -7,4 +7,8 @@ pythonSetupPyPipeline \
   dockerRepo: "python", \
   pypiRepo: "{{ cookiecutter.pypi_repo }}", \
   pypiCredentialsId: "{{ cookiecutter.pypi_credentials_id }} ", \
+  sshAgentUser: "{{ cookiecutter.ci_ssh_agent_user }}", \
```

Comment:
This got removed with #59, so this is not correctly merged. Also I would group your new commands as last one.